templates/frontend/objects/monograph_full.tpl

```diff
@@ -248,13 +248,19 @@
 			{/if}
 
 			{* References *}
-			{if $monograph->getCitations()}
+			{if $parsedCitations->getCount() || $monograph->getCitations()}
 				<div class="item references">
 					<h3 class="label">
 						{translate key="submission.citations"}
 					</h3>
 					<div class="value">
-						{$monograph->getCitations()|nl2br}
+						{if $parsedCitations->getCount()}
+							{iterate from=parsedCitations item=parsedCitation}
+								<p>{$parsedCitation->getRawCitation()|strip_unsafe_html}</p>
+							{/iterate}
+						{elseif $monograph->getCitations()}
+							{translate key="submission.citations"}
```

Comment:
Is this right? It doesn't look like it's printing the raw citations in this case.

Reply:
au... :-O I removed/copied the wrong line... fix coming... THANKS!!!